Matchismo/Matchismo/Model/SetCard.h

```diff
@@ -0,0 +1,28 @@
+//
+//  SetCard.h
+//  Matchismo
+//
+//  Created by Otto Olkkonen on 15/06/2022.
+//
+
+#import "Card.h"
+
+#import <UIKit/UIKit.h>
+
+@interface SetCard : Card
+
+typedef enum {
+  setCardShadingOpen,
+  setCardShadingSolid,
+  setCardShadingStriped,
+  setCardShadingCount
+} SetCardShading;
+
+@property (nonatomic) NSUInteger noOfShapes;
+@property (nonatomic) NSString *shape;
+@property (nonatomic) SetCardShading shading;
+@property (nonatomic) UIColor *color;
```

Comment:
Again here - try to keep UI stuff out of model objects, and try to avoid exposing mutable types publicly.